dev/sparktestsupport/modules.py

```diff
@@ -743,6 +743,17 @@ def __hash__(self):
     ]
 )
 
+docker_integration_tests = Module(
```

Comment:
I thought this test (& the k8s integration one) was intentionally excluded from this file because it is heavy. So, I tried to simply run the docker integration tests via `sbt` in my PR.

Reply:
>I thought this test (& the k8s integration one) was intentionally excluded from this file because it is heavy. 

I think so. So I changed `run-tests.py` not to run the integration tests by default.